gymnasium/vector/vector_env.py

```diff
@@ -10,14 +10,35 @@
 
 
 class VectorEnv(gym.Env):
-    """Base class for vectorized environments. Runs multiple independent copies of the same environment in parallel.
+    """Base class for vectorized environments to runs multiple independent copies of the same environment in parallel.
 
-    This is not the same as 1 environment that has multiple subcomponents, but it is many copies of the same base env.
+    This can provide a linear speed-up in the steps taken per second given the number of sub-environments used.
+    When a sub-environment terminates or truncates, the sub-environment is automatically reset with the final step
+    and info including in the returned info in `"final_observation"` and `"final_info"`. See :meth:`step` for more information.
 
-    Each observation returned from vectorized environment is a batch of observations for each parallel environment.
-    And :meth:`step` is also expected to receive a batch of actions for each parallel environment.
+    The vector environments batch observations, rewards, termination, truncation and info for each parallel environment.
+    In addition, :meth:`step` expects to receive a batch of actions for each parallel environment.
 
-    Notes:
+    Gymnasium contains two types of Vector environments: Asynchronous and Synchronous
+
+    The Vector Environments have the additional attributes for users to understand the implementation
+
+    - :attr:`num_envs` - The number of sub-environment in the vector environment
+    - :attr:`observation_space` - The batched observation space of the vector environment
+    - :attr:`single_observation_space` - The observation space of a single sub-environment
+    - :attr:`action_space` - The batched action space of the vector environment
+    - :attr:`single_action_space` - The action space of a single sub-environment
+
+    Note:
+        The info parameter of :meth:`reset` and :meth:`step` was originally implemented before OpenAI Gym v25 was a list
+        of dictionary for each sub-environment. However, this was modified in OpenAI Gym v25+ and in Gymnasium to a
+        dictionary with a NumPy array for each key. To use the old info style using the :class:`VectorListInfo`.
+
+    Note:
+        To render the sub-environments, use :meth:`call` with "render" arguments. Remember to set the `render_modes`
+        for all the sub-environments during initialization.
+
+    Note:
         All parallel environments should share the identical observation and action spaces.
```

Comment:
Could we put this in quote blocks? Is that a thing in the current build?

Reply:
What do you mean by quotes? https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html